development/compiling/compiling_with_mono.rst

```diff
@@ -136,33 +131,37 @@ Example (Windows)
 ::
 
     # Build temporary binary
-    scons p=windows tools=yes module_mono_enabled=yes mono_glue=no
+    scons p=windows target=editor module_mono_enabled=yes
     # Generate glue sources
-    bin\godot.windows.tools.64.mono --generate-mono-glue modules/mono/glue
+    bin\godot.windows.editor.x86_64.mono --generate-mono-glue modules/mono/glue
+    # Generate binaries
+    ./modules/mono/build_scripts/build_assemblies.py --godot-output-dir=./bin --godot-platform=windows
```

Comment:
Forward slash directory separators should be converted automatically by Windows, but the leading `./` should be removed (it can also be removed on macOS/Linux).